src/lib.rs

```diff
@@ -84,3 +106,44 @@ fn makecircle(centerpoint: Point<f64>, radius: f64, num_vertices: usize) -> Poly
     let polygon = Polygon::new(LineString::from(circle_points), vec![]);
     polygon
 }
+
+// Make a single clock polygon
+pub fn clockpoly(
+    centerpoint: Point<f64>,
+    radius: f64,
```

Comment:
How about `outer_radius` and `inner_radius`?

Reply:
Even better: `radius_outer` and `radius_inner` - they are both types of radius. Updated!